spacecmd/spacecmd.spec

```diff
@@ -24,14 +24,20 @@
 %endif
 
 %if 0%{?fedora} || 0%{?rhel} >= 8
-%{!?pylint_check: %global pylint_check 1}
+%{!?pylint_check: %global pylint_check 0}
```

Comment:
Reason: spacewalk-python3-pylint does not build correctly at CentOS8/RES8/RHEL8 as it requires python3-json (not available) and even if we create it:
```
nothing provides python-astroid, nothing provides python3-astroid, nothing provides python-editdistance, nothing provides python3-editdistance, nothing provides python-isort >= 4.2.5, nothing provides python3-isort >= 4.2.5, nothing provides python-mccabe, nothing provides python3-mccabe, nothing provides python-pytest-runner, nothing provides python3-pytest-runner, nothing provides python-pytest, nothing provides python-six, nothing provides fdupes, nothing provides python2-backports.functools_lru_cache, nothing provides python2-configparser, nothing provides python2-enum34, nothing provides python2-singledispatch, nothing provides python3-tk
```
Fighting this is not needed only for a linter, keeping in mind we'll run it anyway for SLE/openSUSE.